solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/indexing/DocumentIndexingPackageDataLoader.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2008 The University of North Carolina at Chapel Hill
+ * Copyright 2017 The University of North Carolina at Chapel Hill
```

Comment:
I think all the changes to this file (plus a few extra) are already in fcrepo4, which is causing a merge conflict.  So I'm not sure why this is showing up.  Sorry to have to suggest it, but it might be best to either reset this file to fcrepo4's version or start a new branch from fcrepo4 and cherry pick your changes from this branch.